games/chapter2/examples/draw_objects.py

```diff
@@ -0,0 +1,54 @@
+import pygame
+
+pygame.init()
+
+window = pygame.display.set_mode((800, 800))
+pygame.display.set_caption("Drawing and Moving Objects")
+
+BLACK = (0, 0, 0)  # background color
+RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+
+# make a rectangle without the pygame.Rect class
+x = 100  # top-left x value
+y = 400  # top-left y value
+width = 100  # width of the rectangle
+height = 50  # height of the rectangle
+
+# make a pygame.Rect rectangle
+# the syntax is `myvar = pygame.Rect(top-left x, top-left y, width, height)`
+# with 0 as top-left x value, 100 as top-left y value,
+# width = 50, height = 100
+green_rectangle = pygame.Rect(0, 100, 50, 100)
+
+run = True
+while run:
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            run = False
+
+    # move a rectangle that isn't a pygame.Rect object
+    x += 1  # move to the right 1 px
+    y += 1  # move down 1 px
+
+    # move a rectangle that is a pygame.Rect object
+    green_rectangle.move_ip(1, 2)  # moves 1 to the right, 2 down
+    # this is equivalent to green_rectangle = green_rectangle.move(1, 2)
```

Comment:
i ran a quick test, `move_ip()` is not the same as `move()`. the `move()` method moves the rectangle to the specific coordinate, and the `move_ip()` will move the rectangle relative to where it already is. we should change this comment as well as edit the notion to fix that

Reply:
everything else lgtm 👍

Reply:
nvm im a huge dumbass just realized the second example is equating to itself